src/magicbox/ResultPreviewsManager.ts

```diff
@@ -203,8 +203,10 @@ export class ResultPreviewsManager {
     return this.previewsProcessor.processQueries(populateEventArgs.previewsQueries);
   }
 
-  private updateSearchResultPreviewsHeader(text: string) {
-    this.resultPreviewsHeader.text(text);
+  private updateSearchResultPreviewsHeader(suggestion: string) {
+    this.resultPreviewsHeader.setHtml(
+      this.options.previewHeaderText + ' "' + $$('span', { className: 'coveo-omnibox-hightlight' }, suggestion).el.outerHTML + '"'
```

Comment:
Hmm, in the [mockup ](https://coveo.invisionapp.com/share/YGT1U9CAZEM#/screens/374613562)I see the entire text is in bold/highlighted, but based on the screenshot and this line, only the suggestion text is highlighted.

Did UX change their design?

Reply:
I would go like the mockups for a fully bold text

Reply:
My bad, I misinterpreted when you said the text should be in bold. I'll bold the whole thing.